docs/docs/guides/manage/console/instance-settings.mdx

```diff
@@ -87,11 +87,14 @@ The Login Policy defines how the login process should look like and which authen
 
 | Setting                   | Description                                                                                                                                                         |
 | ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Register allowed          | Enable self register possibility in the login ui, this enables username password registration as well as registration with configured external identity providers   |
 | Username Password allowed | Possibility to login with username and password. If this is disabled only login with external identity providers will be allowed                                    |
+| Register allowed          | Enable self register possibility in the login ui, this enables username password registration as well as registration with configured external identity providers   |
 | External IDP allowed      | Possibility to login with an external identity (e.g Google, Microsoft, Apple, etc), If you like to allow external Identity providers add them to the providers list |
-| Force MFA                 | Force a user to register and use a multifactor authentication, Ensure that you have added the MFA methods you want to allow.                                        |
-| Passwordless              | Choose if passwordless login is allowed or not                                                                                                                      |
+| Hide password reset | Disable the self-service option for users to reset their password. |
+| Domain discovery allowed |  |
+| Ignore unknown usernames |  |
```

Comment:
@hifabienne can you please explain what these options do? Just from the labels its not really clear when to use them.